config/routes.rb

```diff
@@ -58,6 +58,7 @@
   patch "users/:user_id/counters/:id", to: "user_counters#update"
   get "users/:id", to: "users#show"
   delete "users/:id", to: "users#destroy"
+  patch "users/me", to: "users#me"
```

Comment:
this should be `users/:id`

`users/me` is a shortcut that resolves to the current users id